src/commands/general/temporaryCommand.ts

```diff
@@ -0,0 +1,177 @@
+import { Colors, EmbedField } from 'discord.js';
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory, Colors as FBWColors } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { getConn } from '../../lib/db';
+import TemporaryCommand from '../../lib/schemas/temporaryCommandSchema';
+
+const helpEmbed = (evokedCommand: String) => makeEmbed({
+    title: 'Temporary Command - Help',
+    description: 'A command to run temporary commands.',
+    fields: [
+        {
+            name: 'Run a Temporary Command',
+            value: makeLines([
+                'To run a temporary command, run the following bot command: ',
```

Comment:
```suggestion
                'To run a temporary command, use the following bot command: ',
```

Double run sounded funky to me?